deploy-infra/deploy-caa.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+set -euo pipefail
+# Check if the DEBUG env var is set to true
+if [ "${DEBUG:-false}" = "true" ]; then
+  set -x
+fi
+
+# Compulsory env vars
+: "${AZURE_RESOURCE_GROUP}:?"
+: "${SSH_KEY}:?"
+
+AZURE_REGION=${AZURE_REGION:-eastus}
+CLUSTER_NAME="${CLUSTER_NAME:-caaEncryptedImages}"
+
+AZURE_SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+AKS_RG="${AZURE_RESOURCE_GROUP}-aks"
+registry="quay.io/confidential-containers"
+AZURE_IMAGE_ID="/CommunityGalleries/cocopodvm-d0e4f35f-5530-4b9c-8596-112487cdea85/Images/podvm_image0/Versions/$(date -v -1d "+%Y.%m.%d" 2>/dev/null || date -d "yesterday" "+%Y.%m.%d")"
```

Comment:
Do you want to pin an image version explicitly for reproducibility of the demo ?